policyengine_us/parameters/gov/states/oh/tax/income/credits/lump_sum_distribution/expected_remaining_years.yaml

```diff
@@ -0,0 +1,491 @@
+description: Ohio provides the following multiples corresponding to the lump sum recipient’s age. 
+metadata:
+  type: single_amount 
+  threshold_unit: year
+  period: year
+  amount_unit: /1
+  label: Ohio lump sum distribution credit multiples
+  reference:
+    - title:  Ohio 2021 Instructions for Filing Original and Amended Individual Income Tax (IT 1040) and  School District Income Tax (SD 100)
```

Comment:
do all of these references point to this table? If not please remove the extra ones